File: app/src/main/java/com/olehliskovych/placenote/di/global/AppModule.java
package com.olehliskovych.placenote.di.global;

import android.content.Context;

import com.olehliskovych.placenote.App;
import com.olehliskovych.placenote.di.scopes.AppScope;

import dagger.Binds;
import dagger.Module;

@Module
public interface AppModule {

    @AppScope
    @Binds
    Context bindContext(App app);
}

File: app/src/main/java/com/olehliskovych/placenote/di/global/AppModule.kt
package com.olehliskovych.placenote.di.global

import android.content.Context
import com.olehliskovych.placenote.App
import dagger.Binds
import dagger.Module

@Module
interface AppModule {

    @Binds
    fun bindContext(app: App): Context


}
